landoapi/models/secapproval.py

```diff
@@ -0,0 +1,47 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+"""Models related to the sec-approval process.
+
+See See https://wiki.mozilla.org/Security/Bug_Approval_Process.
+"""
+
+from sqlalchemy.dialects.postgresql import JSONB
+from sqlalchemy.orm import relationship
+
+from landoapi.storage import db
+
+
+class SecApprovalRevision(db.Model):
```

Comment:
I thought we denormalized just the transaction data table, not the revisions table.  I normalized it  because we have multiple request events per revision.  However the request events table should be small, I don't see a huge performance hit for pulling the revision PHID back into the event table as a column.